lib/bundling-job.js

```diff
@@ -8,12 +8,12 @@ async function runBundlingJob (job, archiveName = 'archive.zip') {
   const files = await findCollectionFileMembers(job.used);
   const archivingParams = files.map((f) => {
     return {
-      path: f.physicalUri.replace('share://', '/share/'),
+      path: f.physicalUri.replace('archive://', '/archive/'),
```

Comment:
Besides that the file-service won't be able to work with `archive://`, here you're using the "Kaleidos files". Those are still using `share://`.